src/rtc/rtc_s1.py

```diff
@@ -295,10 +263,11 @@ def run_parallel(cfg: RunConfig, logfile_path, flag_logger_full_format):
         processing_namespace.apply_absolute_radiometric_correction
 
     # read product path group / output format
-    product_id = cfg.groups.product_group.product_id
-    if product_id is None:
-        product_id = 'rtc_product'
-    product_prefix = f'{product_id}_v{product_version}'
+    runconfig_product_id = cfg.groups.product_group.product_id
+    if runconfig_product_id is None:
+        runconfig_product_id = 'rtc_product'
+    product_id = f'{runconfig_product_id}_v{product_version}'
+    mosaic_product_id = product_id.replace('_{burst_id}', '')
```

Comment:
```suggestion
    mosaic_product_id = product_id.format(burst_id='')
```

Reply:
Here, we do want to include the underscore `_`. So, if `product_id` is `RTC-S1_{burst_id}_S1B` it will become `RTC-S1_S1B`.

Reply:
I see. That being case, please disregard my comment (and the same comment below).